include/sys/syscall_lookup.h

```diff
@@ -261,6 +261,9 @@ SYSCALL_LOOKUP(telldir,                    1)
 
 #if defined(CONFIG_PIPES) && CONFIG_DEV_PIPE_SIZE > 0
   SYSCALL_LOOKUP(nx_pipe,                  3)
+#endif
+
+#if defined(CONFIG_PIPES) && CONFIG_DEV_FIFO_SIZE > 0
   SYSCALL_LOOKUP(nx_mkfifo,                3)
```

Comment:
no real difference?

Reply:
Oops. I did this originally on an older nuttx version, and rebased after that... Didn't notice that it was already fixed. I will just drop this.